src/myuw-help.js

```diff
@@ -36,17 +37,28 @@ class MyUWHelp extends HTMLElement {
   */
   connectedCallback() {
     // Get all attributes
-    this['myuw-help-title']       = this.getAttribute('myuw-help-title');
-    this['open']                  = this.getAttribute('open');
-    this['show-default-content']  = this.getAttribute('show-default-content');
-    this['show-button']       = this.getAttribute('show-button');
-
-    this.$button            = this.shadowRoot.getElementById('help-button');
-    this.$dialog            = this.shadowRoot.getElementById('myuw-help__dialog');
-    this.$dialogTitle       = this.shadowRoot.getElementById('myuw-help__title');
-    this.$backdrop          = this.shadowRoot.getElementById('myuw-help__shadow');
+    this['myuw-help-title'] = this.getAttribute('myuw-help-title');
+    this['open'] = this.getAttribute('open');
+    this['show-default-content'] = this.getAttribute('show-default-content');
+    this['show-button'] = this.getAttribute('show-button');
+
+    this.$button = this.shadowRoot.getElementById('help-button');
+    this.$dialog = this.shadowRoot.getElementById('myuw-help__dialog');
+    this.$dialogContentDefaultElements = this.shadowRoot.getElementById('myuw-help__default-content');
+    this.$dialogContentCustomElements = document.querySelectorAll('myuw-help div');
```

Comment:
Maybe this should be
```javascript
this.$dialogContentCustomElements = document.querySelectorAll('myuw-help [slot=myuw-help-content]');
```
to ensure we are selecting the slotted elements regardless of their tag name? (full disclosure: I didn't test the selector)

Reply:
also, I don't know if there could be a case for multiple myuw-help elements on a page, but if so, this would not seem to distinguish them :/